core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-listing-panel/dot-pages-listing-panel.component.scss

```diff
@@ -1,6 +1,10 @@
 @use "variables" as *;
 
 ::ng-deep {
+    .p-datatable .p-datatable-thead th.p-sortable-column:hover {
```

Comment:
This should go in the global styles and also for not sortables header as well.

Reply:
not sortables columns does not has that bug, only sortables

Reply:
fix moved to global styles